### ISSUE TITLE
blockchain second password

### ISSUE THREAD

**Issue by add1ct3d** [gurnec/btcrecover#260]
can you help us recovering our bitcoins by implementing gpu support for blockchain second password for long passwords?

**Comment by pedrohrcunha** [comment]
It is not possible, by design. Blockchain wallets are protected against GPU runs. I can't explain it technically, but it can't be done.

**Comment by add1ct3d** [comment]
it is already done with me i can bruteforce first password on gpu very fast if you need any help with your wallet or your friend's wallet message me ;)))

**Comment by add1ct3d** [comment]
plus am talking about all versions 1, 2, and 3

**Comment by add1ct3d** [comment]
i am only having problem with implementing second password and i need help in that from gurnec

**Comment by anhday22** [comment]
Currently I have new technology to help open my wallet with a high success rate. If any of your wallet hasn't been successfully opened, please send it to me. I will try to open them.
http://recoverywalletcrypto.com
fb.me/recoverywallet
Thank

**Comment by morarity** [comment]
> i am only having problem with implementing second password and i need help in that from gurnec

python btcrecover.py --data-extract --passwordlist yourpasswordlist.txt --blockchain-secondpass
and then pass the script given in the script in btcrecover for blockchain wallet.
The yourpasswordlist.txt is your candidate potential password.

**Comment by keychainx** [comment]
Yes, it can be done and it has been done. We have a custom made tool that makes about 1-10 million tries per second for blockchain second password on a standard GPU rig. If you dont mind using a beta you can use hashcat beta version.

**Comment by fintray4949** [comment]
> it is already done with me i can bruteforce first password on gpu very fast if you need any help with your wallet or your friend's wallet message me ;)))

How are you my friend I have forgoten my second password  Can you help me  . Alan Fraser 11 Hopton Place . Bundaberg North . 4670 . Queensland .  Australia . fintray63@gmail.com

**Comment by fintray4949** [comment]
Hi My Friend I forgot my secend password at Blockchain can you help

**Comment by psefro** [comment]
I forgot my blockchain second password I been trying to remember or 5 weeks now. So mad. Help.

**Comment by ononbold** [comment]
hi how can i recover second password i forget please help

**Comment by keychainx** [comment]
Read this blog https://keychainx.medium.com/recover-blockchain-com-password-df21cbebac6 or mail me at keychainx@protonmail.com if you need help

**Comment by btcexchanging** [comment]
if you haven't recovered your wallet, i can help you recover your wallet.
I have recovered most of the common wallets and recovery phrases for many users
you can reach me here at my telegram account:  @walletrecover

**Comment by psefro** [comment]
Howdy. No still haven’t damnit

**Comment by anywalletrecovery** [comment]
hi
i might be able to help you recover your password
send me an email here: anywalletrecovery@gmail.com

**Comment by viladmir2022** [comment]
hi, if you still have wallet problem. I can recover your lost wallet.
I have powerful servers and excellent software that can find any password.
and my fee is only 15-25% of the recovered fund
here is my telegram id: @viladmir2022  and Gmail: walletsrecoveryservice@gmail.com

**Comment by johnjames333** [comment]
@psefro 
@ononbold 
@fintray4949 
@add1ct3d 
I have powerful machine to crack all versions of blockchain wallet from 2012-2022 and can recover blockchain password.
I have recovered many blockchain wallets. and some of the had corrupt file. for fast response text me in telegram or gmail
Telegram: @johnjames333

**Comment by doge2021** [comment]
> Hi My Friend I forgot my secend password at Blockchain can you help

 you could contact: [BTC recover](https://twitter.com/Btc2doge) to bruteforce the 2nd password if you still access to the email. there is good chance to recover it. good luck

**Comment by doge2021** [comment]
> I forgot my blockchain second password I been trying to remember or 5 weeks now. So mad. Help.

you could hire: [BTC recover](https://twitter.com/Btc2doge) to bruteforce the 2nd password. there is good chance to recover it. good luck

**Comment by doge2021** [comment]
> hi how can i recover second password i forget please help

it cant be recover by blockchain.com. can only bruteforce it. you can run btc recover by yourself. if you dont know how to do it, can hire [BTC recover](https://twitter.com/Btc2doge)  if you still access to the email there is  chance to recover it

**Comment by fromthemetaverse** [comment]
> can you help us recovering our bitcoins by implementing gpu support for blockchain second password for long passwords?

I can help you then I am here if what I write below has any relevance to your problem you posted a few years ago.

If you have an old blockchain wallet account that keeps asking you for a second password (although you never had any or don't remember putting any) to let you log in and denies you access every time you put any; I can help you access that account if you still have the 1st correct password and know the email address you registered with; You can come to me for help only if you are 200% helpless and clueless and are 200% sure that you can't get help from blockchain.com support or anyone that you know. I can be your last resort and  I will help you have access to the account but for that I will charge you 40% of whatever crypto that is left inside the account. I can prove myself to you if you can provide me any dummy account prior to accessing the one in question. This means you can provide me any blockchain account with zero cryptos inside that is in a similar situation (exact situation of being asked second password and you don't think you put any). I will prove to you that I really can do what I say and let you check out that dummy account first.
Being able to retrieve 60% of whats in a crypto wallet is better than having nothing to retrieve at all from a wallet.

So, Come to me only when you have tried elsewhere and didn't get any result. Like you are totally helpless and tried every method in the book and have nothing to lose ; only then you should come to me and check me out.  Only when you are 200% sure  about it you should come to me.


 I am not asking for any upfront fee from anyone. I do not have any sinister motive. You should try to check my credibility with a dummy blockchain wallet that doesn't hold any crypto inside. I never say that I will be able to pull out a password to a wallet out of nowhere as that can never be achieved as no brains in the universe or a software can do that. What I can do is I can get your access to a wallet back and you will be able to access your account without needing the second password after I am able to crack it.

Remember No brute forcing will let you get thru it.

**Comment by richardsuns** [comment]
@ononbold 
@psefro 
I have already cracked blockchain second password.
I can provide proof.
If still have this problem, I can recover your password.
my telegram id is: @walletrecover
